moveit_ros/planning/planning_request_adapter_plugins/CMakeLists.txt

```diff
@@ -1,13 +1,31 @@
 set(MOVEIT_LIB_NAME moveit_default_planning_request_adapter_plugins)
 
 set(SOURCE_FILES
-  src/empty.cpp
-  src/fix_start_state_bounds.cpp
-  src/fix_start_state_collision.cpp
-  src/fix_start_state_path_constraints.cpp
-  src/fix_workspace_bounds.cpp
-  src/add_time_parameterization.cpp
-  src/add_iterative_spline_parameterization.cpp)
+        src/empty.cpp
+        src/fix_start_state_bounds.cpp
+        src/fix_start_state_collision.cpp
+        src/fix_start_state_path_constraints.cpp
+        src/fix_workspace_bounds.cpp
+        src/add_time_parameterization.cpp
+        src/add_iterative_spline_parameterization.cpp
+        src/chomp_optimizer_adapter.cpp)
+
+find_package(catkin REQUIRED COMPONENTS
+        cmake_modules
+        roscpp
+        moveit_core
+        moveit_msgs
+        pluginlib
+        chomp_motion_planner
+        )
+
+find_package(Eigen3 REQUIRED)
+find_package(Boost REQUIRED)
+
+include_directories(
+        ../../../moveit_planners/chomp/chomp_motion_planner/include/
```

Comment:
Sorry if I missed the discussion, but is relative path a good way to find resource? 

I'm afraid using relative path of another package would limit the portability of a package.

Reply:
I guess with the suggestion i nhttps://github.com/ros-planning/moveit/pull/1012#issuecomment-432568757 we'll be able to replace this hardcoded relative path with auto-substituted path.